doc/userGuide.md

```diff
@@ -309,3 +309,72 @@ openjdk-tests/test-results/openjdk/report
 ```
 
 The JTREG report HTML summary file is then located at `openjdk-tests/test-results/openjdk/report/html/index.html`
+
+## Disable test
+
+#### Automatically exclude a test target
+In the issue that describes the test failure, add a comment with the following format:
+
+```auto exclude test <testName>```
+
+If the testName matches the testCaseName defined in ```<testCaseName>``` element of playlist.xml, the entire test suite will be excluded. If the testName is testCaseName followed by _n, only the (n+1)th variation will be excluded. 
+
+For example:
+
+```
+<test>
+  <testCaseName>jdk_test</testCaseName> 
+    <variations>
+      <variation>NoOptions</variation>
+      <variation>-Xmx1024m</variation>
+    <variations>
+    ...
+```
+To exclude the entire suite:
+
+```auto exclude test jdk_test```
+
+To exclude the test case with variation ```-Xmx1024m```:
+
+```auto exclude test jdk_test_1```
+
+After the comment is left, there will be a auto PR created with the exclude change in the playlist.xml. The PR will be linked to issue. If the testName can not be found in the repo, no PR will be created and there will be a comment left in the issue.
```

Comment:
... a comment left in the issue linking to the failed workflow run for more details.